docs/core/extensions/configuration-providers.md

```diff
@@ -52,13 +52,11 @@ An example *appsettings.json* file with various configuration settings follows:
 
 From the <xref:Microsoft.Extensions.Configuration.IConfigurationBuilder> instance, after configuration providers have been added you can call <xref:Microsoft.Extensions.Configuration.IConfigurationBuilder.Build?displayProperty=nameWithType> to get the <xref:Microsoft.Extensions.Configuration.IConfigurationRoot> object. The configuration root represents the root of a configuration hierarchy. Sections from the configuration can be bound to instances of .NET objects, and later provided as <xref:Microsoft.Extensions.Options.IOptions%601> through dependency injection.
 
-Consider the `TransientFaultHandlingOptions` record type defined as follows:
```

Comment:
I don't feel "object" is the correct term. It implies that it's an *instance* of a type, while it's the type itself.

Reply:
Everything in .NET is an `object`. I describe it as the object's definition, both a record and a class are an object. I don't talk about instantiating anything. I've updated this regardless, but don't think it's necessary.